### ISSUE TITLE
Component List

### ISSUE THREAD

**Issue by kinduff** [stecman/hw-boot-selection#3]
Would love to build one of these but I'm missing the components you're using. What are the models? Or names?

**Comment by stecman** [comment]
Hey @kinduff,

I've just updated the code to build for the STM32F103 by default, since that's a fairly common part - used on the classic "blue pill" and "maple" STM32 boards for example. Any STM32 part with USB support should work, though minor additions to the code will be needed to support series other than the F0 and F1.

libopencm3, which is doing most of the heavy lifting here, supports [quite a range of devices](https://github.com/libopencm3/libopencm3#readme). The USB portion of the code doesn't use anything STM32 specific, so with some work it should in `main.c` it should also be possible to run on a SAMD21 and other parts.

Other than the switch, there are no external parts. The switch connects pin A6 to GND when on, and when off pin A6 is pulled high.

I've updated the Hackaday project page with a parts list which is an abbreviated version of this reply.

Is that enough to get you started?
